docs/release_new_version.md

```diff
@@ -0,0 +1,98 @@
+# Release new site version
+
+This doc covers the process of releasing a new build version to the live site.
+
+First of all, note that each commit generates a build artifact with an id of `gh-xxxx`.
+It's important to first wait until the CI has generated such artifact, and then find the id of your target commit.
+(It will be tagged to your target commit automatically)
+
+## Ensure your build has been successfully uploaded by CI
+
+Login into the admin node with `ce admin login`, and once there remember to connect to the tmux session,
+with `tmux at` (`Ctrl+B, D` by default to later detach instead of closing the connection).
+
+Now run `ce --env staging builds list` and make sure that your target `gh-xxxx` build is present.
+
+_If this is not the case, make sure that there were no errors in the CI, or ask around for help._
+
+## Run compiler discovery of your target build
+
+Run a "Compiler Discovery" workflow in https://github.com/compiler-explorer/infra/actions/workflows/compiler-discovery.yml
+
+You need to fill 3 inputs:
+ - **Environment**: This is usually `staging`, but can be run for `beta` and `prod` too
+ - **Branch**: The branch that your target commit belongs to. Usually `main`
+ - **Build number**: The `gh-xxxx` id you identified
+
+It usually takes less than 5 min. for the discovery to run.
```

Comment:
Can also run this from the cmdline (though not the admin node at least not yet):
```
gh workflow run -R compiler-explorer/infra 'Compiler discovery' -f buildnumber=gh-xxxx
```